src/Decompressor/DecompressorSnappy.php

```diff
@@ -16,16 +16,25 @@ public function decompress(SplFileInfo $sourceFile): void
         try {
             $baseName = $sourceFile->getBasename('.snappy');
             $destinationPath = $this->getDestinationPath($sourceFile);
-            (new Process(
-                'python3 -m snappy -d ' . escapeshellarg($sourceFile->getPathname()) . ' ' . escapeshellarg($destinationPath . '/'  .$baseName)
-            ))
-                ->setTimeout(null)
+            $process = new Process([
+                'python3',
+                '-m',
+                'snappy',
+                '-d',
+                $sourceFile->getPathname(),
+                $destinationPath . '/' . $baseName,
+            ]);
+            $process->setTimeout(null)
                 ->setIdleTimeout(null)
                 ->mustRun();
         } catch (ProcessFailedException $e) {
-            throw new UserException(
```

Comment:
tohle bohužel může zachytit i jiný chyby, nešlo by je třeba vyjmenovat?

Reply:
mas na mysli neco konkretniho? nestudovali jsme to ani predtim proc to failuje, proste nejde rozbalit.

Reply:
jsem teď ponořenej do toho OOM, takže to může bejt taky jeden z důvodů, proč procesor může sletět. asi bych vzal jen hlášky, co ti to hodí v testech a reagoval přes nějakej regexp jen na ně.

Reply:
no ja nevim a kdyz to neodpovida nejakymu regexpu tak co? application error? user error? mam dost strach z toho ze si zavedem kupu app erroru